build_tools/docker/image_graph.json

```diff
@@ -0,0 +1,126 @@
+{
+  "base": {
+    "deps": [],
+    "digest": "sha256:7dbb7e97e0baa6d4512822b5cd4f601d840a6f950f67c2df497a24cae64a0595",
+    "url": "gcr.io/iree-oss/base@sha256:7dbb7e97e0baa6d4512822b5cd4f601d840a6f950f67c2df497a24cae64a0595"
+  },
```

Comment:
This file is now updated both manually and by automation. I'd prefer to keep those separate, to make it clear what can be edited and when (otherwise you could have local changes to the graph structure pending in your editor when the script runs and changes the digests, for example).

Reply:
I guess that's technically true for other files that use the Docker image digests/sha values, but this file in particular could be touched by both while making changes to the Docker file definitions.

Reply:
Updated. Now `image_deps.json` only describes the dependencies and keep prod_digests.txt to record the digests.